.k8/helm3/ssv-liquidator/templates/liquidator-mainnet-1.yml

```diff
@@ -2,40 +2,40 @@
 apiVersion: v1
 kind: Service
 metadata:
-  name: ssv-liquidator-worker-7-svc
+  name: liquidator-mainnet-1-svc
   namespace: {{ .Values.namespace }}
   labels:
-    name: ssv-liquidator-worker-7-svc
+    name: liquidator-mainnet-1-svc
 spec:
   type: {{ .Values.service.type }}
   ports:
     - port: {{ .Values.service.port }}
       protocol: TCP
       targetPort: 3000
   selector:
-    app: ssv-liquidator-worker-7
+    app: liquidator-mainnet-1
 ---
 apiVersion: apps/v1
 kind: Deployment
 metadata:
   labels:
-    app: ssv-liquidator-worker-7
-  name: ssv-liquidator-worker-7
+    app: liquidator-mainnet-1
+  name: liquidator-mainnet-1
   namespace: {{ .Values.namespace }}
 spec:
-  replicas: {{ .Values.replicaCount.ssv_liquidator_worker_7 }}
+  replicas: {{ .Values.replicaCount.liquidator_mainnet_1 }}
   selector:
     matchLabels:
-      app: ssv-liquidator-worker-7
+      app: liquidator-mainnet-1
   strategy:
     type: Recreate
   template:
     metadata:
       labels:
-        app: ssv-liquidator-worker-7
+        app: liquidator-mainnet-1
     spec:
       containers:
-      - name: ssv-liquidator-worker-7
+      - name: liquidator-mainnet-1
```

Comment:
@systemblox we need to have for each deployment its own secret for account private key:
in total 5 (1 for j1, 1 for j2 and 3 for mainnet)- and also lets have the secret names also if name convention please